.wsignore

```diff
@@ -19,6 +19,7 @@ acceptance/selftest/record_cloud/volume-io/hello.txt
 
 # "bundle run" has trailing whitespace:
 acceptance/bundle/integration_whl/*/output.txt
+acceptance/bundle/run/basic/output.txt
```

Comment:
why is it here? The last command is "echo" which includes a newline.

Reply:
Is this easily fixable? I would prefer to keep this ignore list to the absolute minimum.

Reply:
Trying to fix in https://github.com/databricks/cli/pull/2864